{"name": "Тест загрузки МК", "status": "broken", "statusDetails": {"message": "Exception: Элемент не найден", "trace": "self = <pages.sign_in_page.SignInPage object at 0x00000275DBAC7CD0>\nlocator = ('css selector', '#root > div > div:nth-child(3) > div > div > div:nth-child(1) > div > div > div > div> div > div.Mui...MuiCollapse-vertical.MuiCollapse-entered.css-c4sutr > div > div > div > div > ul > div:nth-child(7) > div > li > span')\n\n    def text_of_visible_element(self, locator):\n        try:\n>           element = Wait(self.driver, 5).until((ec.visibility_of_element_located(locator)))\n\n..\\pages\\base_page.py:90: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"8c921bf4d6c46ca52fe04c4a91215292\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x00000275DB739DA0>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       Backtrace:\nE       \t(No symbol) [0x00BC37D3]\nE       \t(No symbol) [0x00B58B81]\nE       \t(No symbol) [0x00A5B36D]\nE       \t(No symbol) [0x00A8D382]\nE       \t(No symbol) [0x00A8D4BB]\nE       \t(No symbol) [0x00AC3302]\nE       \t(No symbol) [0x00AAB464]\nE       \t(No symbol) [0x00AC1215]\nE       \t(No symbol) [0x00AAB216]\nE       \t(No symbol) [0x00A80D97]\nE       \t(No symbol) [0x00A8253D]\nE       \tGetHandleVerifier [0x00E3ABF2+2510930]\nE       \tGetHandleVerifier [0x00E68EC1+2700065]\nE       \tGetHandleVerifier [0x00E6C86C+2714828]\nE       \tGetHandleVerifier [0x00C73480+645344]\nE       \t(No symbol) [0x00B60FD2]\nE       \t(No symbol) [0x00B66C68]\nE       \t(No symbol) [0x00B66D4B]\nE       \t(No symbol) [0x00B70D6B]\nE       \tBaseThreadInitThunk [0x75A800F9+25]\nE       \tRtlGetAppContainerNamedObjectPath [0x77B77BBE+286]\nE       \tRtlGetAppContainerNamedObjectPath [0x77B77B8E+238]\n\n..\\..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-eUzg4zEH-py3.11\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:95: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <test_calculation_constructor.TestCalculationConstructor object at 0x00000275DB715810>\nsign_in_to_stand = <pages.sign_in_page.SignInPage object at 0x00000275DBAC7CD0>\nmb_name = 'МК_Upstream_v22.6Б_6+6_ЮУНГ_Бурение_3_51', clear_after_upload = True\n\n    @allure.feature(\"Загрузка МК на фронт\")\n    @allure.story(\"Загрузка всех имеющихся МК на стенд и проверка, что все загрузилось корректно\")\n    @allure.title(\"Тест загрузки МК\")\n    @allure.severity(allure.severity_level.BLOCKER)\n    @allure.description(f\"Тест запустил - {tester_name}. \\nСтенд, на котором запускался тест - {current_stand}\\nДата \"\n                        f\"запуска: {dt.now()}\")\n    @pytest.mark.calculacion_constructor\n    # @pytest.mark.parametrize('mb_name', [CCFiles.MB_GEE_filename, CCFiles.MB_KUV_filename, CCFiles.MB_CNT_filename,\n    #                                      CCFiles.MB_YUUNG_filename, CCFiles.MB_YAG_filename,\n    #                                      CCFiles.MB_YUUNG_BUR_filename])\n    @pytest.mark.parametrize('mb_name', [CCFiles.MB_GEE_filename,\n                                         CCFiles.MB_YUUNG_BUR_filename])\n    def test_parser(self, sign_in_to_stand, mb_name, clear_after_upload=True):\n        with allure.step(\"Прохождение авторизации на стенде\"):\n            page = sign_in_to_stand\n        with allure.step(\"Переход в конструктор расчетов\"):\n            CCPage.move_to_calculation_constructor_page(page)\n            sleep(1)\n            CCPage.is_page_calculation_page(page)\n        if clear_after_upload:\n            with allure.step(\"Очистка мастер-книг со стенда\"):\n                CCPage.delete_all_masterbooks(page)\n        with allure.step(f'Загрузка мастер-книги на стенд'):\n>           CCPage.upload_mb(page, mb_name)\n\ntest_calculation_constructor.py:53: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\pages\\calculation_constructor_page.py:170: in upload_mb\n    CalculationConstructorPage.mb_checker(self, cons_list[j], cases_list[j])\n..\\pages\\calculation_constructor_page.py:129: in mb_checker\n    case_name = self.text_of_visible_element(case_locator)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.sign_in_page.SignInPage object at 0x00000275DBAC7CD0>\nlocator = ('css selector', '#root > div > div:nth-child(3) > div > div > div:nth-child(1) > div > div > div > div> div > div.Mui...MuiCollapse-vertical.MuiCollapse-entered.css-c4sutr > div > div > div > div > ul > div:nth-child(7) > div > li > span')\n\n    def text_of_visible_element(self, locator):\n        try:\n            element = Wait(self.driver, 5).until((ec.visibility_of_element_located(locator)))\n            return element.text\n        except TimeoutException:\n>           raise Exception(\"Элемент не найден\")\nE           Exception: Элемент не найден\n\n..\\pages\\base_page.py:93: Exception"}, "description": "Тест запустил - Данил Подольный. \nСтенд, на котором запускался тест - http://demo.rospan.iifa.ru/economics/\nДата запуска: 2023-03-09 12:47:18.891046", "steps": [{"name": "Прохождение авторизации на стенде", "status": "passed", "start": 1678341197485, "stop": 1678341197485}, {"name": "Переход в конструктор расчетов", "status": "passed", "start": 1678341197485, "stop": 1678341198855}, {"name": "Очистка мастер-книг со стенда", "status": "passed", "steps": [{"name": "Подчет всех МК на стенде", "status": "passed", "start": 1678341199086, "stop": 1678341204984}, {"name": "Удаление всех МК на стенде", "status": "passed", "start": 1678341204984, "stop": 1678341253147}, {"name": "Проверка остались ли МК на странице", "status": "passed", "start": 1678341253147, "stop": 1678341258302}, {"name": "F5 и повторная проверка", "status": "passed", "start": 1678341258302, "stop": 1678341264756}], "start": 1678341198855, "stop": 1678341264756}, {"name": "Загрузка мастер-книги на стенд", "status": "broken", "statusDetails": {"message": "Exception: Элемент не найден\n", "trace": "  File \"C:\\Users\\dpodolny\\PycharmProjects\\DatcyEstateAT_v.0.1\\IIFA_Economics_Auto-tests_v.0.3\\tests\\test_calculation_constructor.py\", line 53, in test_parser\n    CCPage.upload_mb(page, mb_name)\n  File \"C:\\Users\\dpodolny\\PycharmProjects\\DatcyEstateAT_v.0.1\\IIFA_Economics_Auto-tests_v.0.3\\pages\\calculation_constructor_page.py\", line 170, in upload_mb\n    CalculationConstructorPage.mb_checker(self, cons_list[j], cases_list[j])\n  File \"C:\\Users\\dpodolny\\PycharmProjects\\DatcyEstateAT_v.0.1\\IIFA_Economics_Auto-tests_v.0.3\\pages\\calculation_constructor_page.py\", line 129, in mb_checker\n    case_name = self.text_of_visible_element(case_locator)\n                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\dpodolny\\PycharmProjects\\DatcyEstateAT_v.0.1\\IIFA_Economics_Auto-tests_v.0.3\\pages\\base_page.py\", line 93, in text_of_visible_element\n    raise Exception(\"Элемент не найден\")\n"}, "start": 1678341264756, "stop": 1678341311341}], "attachments": [{"source": "cb62eaca-b03e-4b26-bf58-efb8778ca5c2-attachment.mp4", "type": "video/mp4"}, {"name": "stdout", "source": "5a6770b7-677e-45aa-8989-bf81a92d0ecd-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "mb_name", "value": "'МК_Upstream_v22.6Б_6+6_ЮУНГ_Бурение_3_51'"}], "start": 1678341197171, "stop": 1678341312841, "uuid": "f129bed8-e6c0-4782-b28f-992437a43d8a", "historyId": "1f002515f30fbf7c7f659df4415c1ef0", "testCaseId": "649e79659ec5cf1411c309c586996b64", "fullName": "tests.test_calculation_constructor.TestCalculationConstructor#test_parser", "labels": [{"name": "feature", "value": "Загрузка МК на фронт"}, {"name": "story", "value": "Загрузка всех имеющихся МК на стенд и проверка, что все загрузилось корректно"}, {"name": "severity", "value": "blocker"}, {"name": "tag", "value": "calculacion_constructor"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_calculation_constructor"}, {"name": "subSuite", "value": "TestCalculationConstructor"}, {"name": "host", "value": "C100119"}, {"name": "thread", "value": "2508-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_calculation_constructor"}]}